Add cotes de credit data

bboulesteix opened the pull request:
This pull request adds data about the cote de credit granted to some companies by the Banque de France. This is a follow up to the publication of [our magazine article](https://memo.bank/magazine/covid-change-credit-entreprises/) on that topic.

bboulesteix committed:
Add cotes de credit data

This commit adds data about the cote de credit granted to some companies by the Banque de France.

bboulesteix committed:
Add separator type

This commit details the type of separator used to build the CSV file.

dirtyhenry commented on banque-de-france-cotes-de-credit/README.md:
```diff
@@ -0,0 +1,34 @@
+| `6`               | Nombre d’entreprises ayant reçu une cote de crédit de 6 (très faible)                                           |
+| `7`               | Nombre d’entreprises ayant reçu une cote de crédit de 7 (au moins un incident de paiement)                      |
+| `8`               | Nombre d’entreprises ayant reçu une cote de crédit de 8 (menacée)                                               |
+| `9`               | Nombre d’entreprises ayant reçu une cote de crédit de 3++ (compromise)                                          |
```
Typo ? À mon avis ce devrait être : 

```
Nombre d’entreprises ayant reçu une cote de crédit de 9 (compromise)
```

Non ?

bboulesteix replied:
En effet, bien vu @dirtyhenry. Je corrige, merci.

dirtyhenry commented on banque-de-france-cotes-de-credit/README.md:
```diff
@@ -0,0 +1,34 @@
+Pour les années précédentes, se reporter aux versions archivées de la page FIBEN :
+
+- [2020](https://entreprises.banque-france.fr/sites/default/files/media/2021/01/21/fiben_cotation_chiffres_cles.pdf)
+- [2018](https://web.archive.org/web/20200422083116/https://www.fiben.fr/presentation/lessentiel-de-fiben)
+- [2019](https://www.fiben.fr/sites/bdf_fiben/files/medias/fiben_2019.png)
```
Tri à réparer ?

bboulesteix replied:
Oui, c’est un peu moche, tu as raison. Je corrige, merci.

bboulesteix committed:
Edit for clarity and order

This commit fixes a typo in the description of the CSV. It also corrects the order of appearance of the years listed in our unordered list.
